server/game/cards/03_TWI/units/EncouragingLeadership.ts

```diff
@@ -0,0 +1,24 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { WildcardZoneName } from '../../../core/Constants';
+
+export default class EncouragingLeadership extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '4916334670',
+            internalName: 'encouraging-leadership'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Give each friendly unit +1/+1 for this phase.',
+            immediateEffect: AbilityHelper.immediateEffects.forThisPhaseCardEffect((context) => ({
+                target: context.source.controller.getUnitsInPlay(WildcardZoneName.AnyArena, (card) => card.isUnit()),
```

Comment:
The card.isUnit() check is redundant, `getUnitsInPlay` already filters out everything that isn't a unit. So you can just remove that argument.